creator-node/src/components/replicaSet/exportComponentService.js

```diff
@@ -99,7 +99,6 @@ const exportComponentService = async ({
       const curCnodeUserClockVal = cnodeUser.clock
 
       // Resets cnodeUser clock value to requestedClockRangeMax to ensure consistency with clockRecords data
-      // Also ensures secondary knows there is more data to sync
       if (cnodeUser.clock > requestedClockRangeMax) {
         // since clockRecords are returned by clock ASC, clock val at last index is largest clock val
         cnodeUser.clock = requestedClockRangeMax
```

Comment:
how do you feel about not assigning this to cnodeUser.clock and instead using a new variable. the first i looked at this code after a while i was worried this was updating the cnodeUser object by accident cause it looks like it updates the result of the sequelize query

Reply:
the only thing that would change is the logs below would reference this new var instead of cnodeUser.clock

Reply:
i dont think i follow - whats your rec?

Reply:
fixed